### DISCUSSION TITLE
Error de toker '&&' en la creacion del tag para TP macowins

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#398] [Consultas generales] [answered]
Buenas Tardes!

Me encontraba terminando el tp individual de macowins y surgio el siguiente error a la hora de crear el tag.

Al realizar el comando: 
`mvn clean verify && git tag entrega-final && git push origin HEAD --tags`
Me devuelve el siguiente error:

```
El token '&&' no es un separador de instrucciones válido en esta versión.
En línea: 1 Carácter: 43
+ mvn clean verify && git tag entrega-final && git push origin HEAD --t ...
+                                           ~~
El token '&&' no es un separador de instrucciones válido en esta versión.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine
```

Por lo que entiendo el terminal no reconoce al operador && de instrucciones, pero dado que estoy utilizando el comando que se me brindo de repositorio, quería consultar si es normal que me aparezca este error.

¿Queria consultar si realizo los comandos por separado uno después del otro: 
`mvn clean verify`
 `git tag entrega-final`
`git push origin HEAD --tags`
EL resultado es el mismo?

Aclaro que anteriormente subí mal un tag pero este ya fue eliminado tanto localmente como en el repositorio utilizando los comandos brindados en el foro

Espero su respuesta, saludos

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola! La línea propuesta es para sistemas unix (Mac, Linux) o emuladores (cygwin, git-bash). Nunca ví ese error pero sospecho que estás usando PowerShell (es decir, la consola estándar de Windows), que no es compatible en general con los instructivos que proveemos. ¿Podrías por favor confirmar esto?

En general te sugeriría no usar PowerShell o similar, sino alguna de las opciones anteriores. 

Más allá de eso, el operador `&&` ejecuta un comando y a continuación **y solo si el primero tuvo éxito**, el segundo. Podés ejecutar los comandos uno a continuación del otro pero vas a tener que prestar atención a que ninguno falle antes de ejecutar el siguiente.

¡Saludos!

**Reply by tobiascastaneda2000** [reply]
Buenas Tardes profesor!!

Muchísimas gracias por las ayuda. 

Lo que menciono, estaba utilizando Windows PowerShell por defecto para realizar los comandos. 
Cambie la terminal a Command Prompt y me acepto el comando perfectamente.

Saludos!!
